helm-gtags.el

```diff
@@ -745,10 +749,11 @@ Always update if value of this variable is nil."
 (defun helm-gtags--parse-file-candidate-transformer (file)
   (let ((removed-file (replace-regexp-in-string "\\`\\S-+ " "" file)))
     (when (string-match "\\`\\(\\S-+\\) \\(\\S-+\\) \\(.+\\)\\'" removed-file)
-      (format "%-25s %-5s %s"
-              (match-string-no-properties 1 removed-file)
-              (match-string-no-properties 2 removed-file)
-              (match-string-no-properties 3 removed-file)))))
+      (setq helm-gtags--last-input (match-string 1 removed-file))
+      (format "%s:%s:%s"
```

Comment:
Seems the original format ("%-25s %-5s %s") is better-looking than your format here.